### ISSUE TITLE
Can't clone solito 

### ISSUE THREAD

**Issue by Elie-A-98** [nandorojo/solito#487]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Do you want this issue prioritized?

- [ ] Yes, I have sponsored
- [ ] Not urgent

### Current Behavior

Following the docs to clone a minimal starter project `npx create-solito-app@latest my-solito-app` is throwing this error

`> yarn install
Usage Error: Unsupported package manager specification (bun@1.1.18)`

### Expected Behavior

To clone successfully

### Versions

```markdown
- Solito: 4.2.2
- Next.js: -
- Expo: -
- React Native: -
```

**Comment by Autsider666** [comment]
+1, can't install either the default or the `with-tailwind` alternative + both crash during build when using the deploy button in the documentation:
![image](https://github.com/user-attachments/assets/37513ac3-d798-4217-9618-0c41e488108c)

**Comment by jeffreyvalle** [comment]
> +1, can't install either the default or the `with-tailwind` alternative + both crash during build when using the deploy button in the documentation: ![image](https://private-user-images.githubusercontent.com/2787376/356823096-37513ac3-d798-4217-9618-0c41e488108c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMzMzMzODksIm5iZiI6MTcyMzMzMzA4OSwicGF0aCI6Ii8yNzg3Mzc2LzM1NjgyMzA5Ni0zNzUxM2FjMy1kNzk4LTQyMTctOTYxOC0wYzQxZTQ4ODEwOGMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDgxMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA4MTBUMjMzODA5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZGNlNjUwOTlkYWFkM2QxN2FjZGNlNDQ1MDA2YWM4MDAwN2U2ZTQzNTE0YWQ5YjM3NGExZjdmYmI5YWRjYTAzMCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.qXxNUlPhtpILzwA_Sr1PS4MiZGz7w4gdWRS9jrvMO1k)

in project root in package.json, try changing ```"packageManager": "bun@1.1.18"``` to ```"packageManager": "yarn@3.4.1"``` as a temp fix and see if it works. This allowed me to yarn install and build. Using the CLI to install the starter monorepo is still mostly working, it will clone the repo to your machine but will break after cd into the new directory it creates for your project because it tries to yarn install. After you run the CLI the first time, make the change I described to package.json in root and see if you have any luck.

**Comment by Elie-A-98** [comment]
@jeffreyvalle but u cannot use the cli to install. You can just clone the blank ans try. I did and it worked

**Comment by jeffreyvalle** [comment]
@Elie-A-98 thanks, you're correct after testing further my suggestion does not fix anything :)

**Comment by SimonHudec** [comment]
it's far more broken than this, even if you manage to fix the bun packager, be ready for another round of errors

**Comment by nandorojo** [comment]
I think this was caused by #480

**Comment by Elie-A-98** [comment]
> I think this was caused by #480

Yes. Apart from the version bumps are there reasons for replacing yarn with bun ? Or just for faster builds ?

Cz patch-package does not work with bun. I used bun patch and had to move to linux because it had some pernission issues on windows

**Comment by nandorojo** [comment]
We should switch this back to yarn 4 I think

**Comment by tremby** [comment]
Additionally, with `--use-npm` (which I prefer), there's this:

```
$ npx create-solito-app@latest my-solito-app -t with-tailwind --use-npm
🧑🎤 Creating solito app...

Creating a new solito app in my-solito-app...
my-solito-app folder created.
Copying template into my-solito-app...

Downloaded template into my-solito-app...

my-solito-app created!
Installing packages. This might take a couple of minutes.

> npm install
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: expo-app@1.0.0
npm ERR! Found: expo@51.0.28
npm ERR! node_modules/expo
npm ERR!   expo@"^51.0.23" from expo-app@1.0.0
npm ERR!   apps/expo
npm ERR!     expo-app@1.0.0
npm ERR!     node_modules/expo-app
npm ERR!       workspace apps/expo from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer expo@"^49.0.0" from expo-router@2.0.15
npm ERR! node_modules/expo-router
npm ERR!   expo-router@"^2.0.5" from expo-app@1.0.0
npm ERR!   apps/expo
npm ERR!     expo-app@1.0.0
npm ERR!     node_modules/expo-app
npm ERR!       workspace apps/expo from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR!
npm ERR! For a full report see:
npm ERR! /home/me/.npm/_logs/2024-08-15T23_03_28_038Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in: /home/me/.npm/_logs/2024-08-15T23_03_28_038Z-debug-0.log

[solito] error installing node_modules with npm
 npm exited with non-zero code: 1

```

**Comment by nandorojo** [comment]
Hey guys, can you see if this is fixed? Thanks for reporting
